server/handlers/meshsync_handler.go

```diff
@@ -116,7 +116,10 @@ func (h *Handler) GetMeshSyncResources(rw http.ResponseWriter, r *http.Request,
 	}
 
 	if search != "" {
-		result = result.Where("kubernetes_resource_object_meta.name LIKE ?", "%"+search+"%")
+
+		result = result.
+			Joins("JOIN kubernetes_resource_object_meta ON kubernetes_resource_object_meta.id = kubernetes_resources.id").
```

Comment:
No I have tried doing that.

We cannot use `kubernetes_resource_object_meta` in where clause to perform LIKE based filtering when we do just Preload. 
Moreover, using `db.Where(&KubernetesResourceObjectMeta{Name: "%"+search"%"})` converts to `kubernetes_resource_object_meta.name = ?` and doesn't uses LIKE.

So, join is required so that we can easily using the LIKE while filtering.